spring-boot-examples/producer-app/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same as other POM, the formatting is off here

Reply:
done